shortcuts.yaml

```diff
@@ -1,26 +1,29 @@
 short options:
-'-t': '--authType, --targetTypes, --to, --type, --title, --fieldTitle, --targetUrl, --listTitle, --sectionTemplate, --text, --webTemplate'
-'-u': '--userName, --usageGuidelinesUrl, --appCatalogUrl, --webUrl, --url, --siteUrl'
-'-p': '--password, --path, --properties, --filePath, --policy, --clientSideComponentProperties, --pageSize, --parentFolderUrl, --principals, --promoteAs, --selectProperties, --parentWebUrl'
+'-t': '--authType, --targetTypes, --to, --title, --targetUrl, --sectionTemplate, --text'
+'-u': '--userName, --usageGuidelinesUrl, --appCatalogUrl, --url'
+'-p': '--password, --path, --properties, --filePath, --policy, --clientSideComponentProperties, --parentFolderUrl, --principals, --promoteAs, --parentWebUrl'
 '-c': '--certificateFile, --packageCreatedBy, --classifications, --channelId, --contentTypeId, --clientSideComponentId, --contentType, --clientState, --column, --classification, --content, --comment'
-'-i': '--clientId, --grantId, --appId, --id, --templateId, --groupId, --teamId, --position, --hubSiteId, --includeAssociatedSites, --listId, --requestId, --siteDesignId, --runId, --taskId'
+'-i': '--clientId, --grantId, --id, --templateId, --teamId, --position, --hubSiteId, --includeAssociatedSites, --requestId, --siteDesignId, --runId, --taskId'
 '-r': '--resourceId, --role, --origin, --required, --rights, --resource'
-'-s': '--scope, --packageSourceEnvironment, --subject, --siteUrl, --sortOrder, --sourceUrl, --pageSize, --systemUpdate, --section, --siteScripts'
-'-n': '--displayName, --name, --packageDisplayName, --userName, --notificationUrl, --pageNumber, --pageName'
+'-s': '--packageSourceEnvironment, --subject, --siteUrl, --sourceUrl, --systemUpdate, --section, --siteScripts'
+'-n': '--displayName, --name, --packageDisplayName, --notificationUrl, --pageNumber, --pageName'
 '-e': '--environment, --enabled, --commandUIExtension, --expirationDateTime'
-'-d': '--packageDescription, --description, --displayName, --defaultClassification, --date'
-'-f': '--format, --flow, --fieldId, --filter, --folder, --fileUrl, --folderUrl, --forceRefresh, --fields'
+'-d': '--packageDescription, --description, --defaultClassification, --date'
+'-f': '--format, --flow, --fieldId, --folder, --fileUrl, --folderUrl, --forceRefresh, --fields'
 '-m': '--mailNickname, --messageId, --previewImageUrl'
```

Comment:
Why `-m` for `previewImageUrl`?

Reply:
That was already there. I just have not removed from the list. We can remove it.

Reply:
Will move it under `-p` and we will decide later if needs shortcut.